### PR TITLE
Added a contrib section with a sample docker-compose file

### PR DESCRIPTION
**Pull request by aram535**
Fully tested complete docker-compose and Dockerfile to download and build three containers ES, KB, FSCrawler.

To enable, edit the docker-compose and change the path_to_files_to_scan to the actual path of files to be scanned.  If need be, change the Dockerfile to match the version that you want to grab from github.

### CODE REVIEW

**Comment by aram535** [comment]
If you would tell me where you like to have it, I'll be happy to add a doc section as a new PR.

**Comment by dadoonet** [comment]
Ideally you can add a new commit to this PR and add some content in https://github.com/dadoonet/fscrawler/blob/master/docs/source/user/tips.rst

Thanks !

**Comment by aram535** [comment]
committed.

File: contrib/docker-compose-example/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    ports:
+      - 5601:5601
+    environment:
+      ELASTICSEARCH_URL: http://es01:9200
+      ELASTICSEARCH_HOSTS: http://es01:9200
+    restart: always
+    networks:
+      - fscrawler_net
+
+  fscrawler:
```
Comment (dadoonet) [inline contrib/docker-compose-example/docker-compose.yml:59]: That's something that I will modify once we merge the production of the Docker image in dockerhub. #820

Comment (aram535) [reply contrib/docker-compose-example/docker-compose.yml:59]: Yeah once it's in Dockerhub it'll be a lot easier.  No build step needed at all for most people, some still like to build their own but most will just need the docker-compose file.

**Comment by k3ninho** [comment]
Doesn't work for me: the ENV lines duplicate SNAPSHOT as per...

diff --git a/contrib/docker-compose-example/Dockerfile-fscrawler b/contrib/docker-compose-example/Dockerfile-fscrawler
index 8d02d38..33132ca 100644
--- a/contrib/docker-compose-example/Dockerfile-fscrawler
+++ b/contrib/docker-compose-example/Dockerfile-fscrawler
@@ -11,7 +11,7 @@ FROM openjdk:8-jre-alpine
 
 ENV ES es7
 ENV VERSION 2.7-SNAPSHOT
-ENV FSCRAWLER_FNAME fscrawler-$ES-$VERSION-SNAPSHOT
+ENV FSCRAWLER_FNAME fscrawler-$ES-$VERSION
 
 RUN apk add --update \
  && addgroup -S fscrawler && adduser -S -G fscrawler fscrawler

**Comment by aram535** [comment]
Thanks.  If you use 2.7 for VERSION, it'll work fine. this is in contrib, which means it is a template for you to copy and use.   My thinking was that its for 'master' which means it'll always (it is an assumption) be a SNAPSHOT of the next version.
I do agree that it's probably a good idea to change it.  Why not submit it as a pull request rather than just a diff?

**Pull request merged**